### PR TITLE
Chore/fix testdata script

### PR DESCRIPTION
**Pull request by ds-jhartmann**

### CODE REVIEW

File: local/testing/testdata/README.md [hunk 1]
```diff
@@ -140,8 +140,17 @@ twin with this BPN will be created with an invalid endpointAddress.
 the testdata file and pass the id with this parameter. If nothing is provided, the default policy will be empty with the
 id "default-policy".
 
-**-bpns, --bpns <[BPNS ...] >**  
+**-bpns, --bpns <[BPN ...] >**  
 (Optional) A list of BPNs Filter upload to upload only specific BPNs
 
+**-d, --dataplane <[URL ...] >**  
+(Optional) If specified, these public datalane URLs will be displayed in the digital twin href
```
Comment (mkanal) [inline local/testing/testdata/README.md:147]:
```suggestion
(Optional) If specified, these public dataplane URLs will be displayed in the digital twin href
```

Comment (mkanal) [inline local/testing/testdata/README.md:147]:
```suggestion
(Optional) If specified, these public datalane URLs will be displayed in the digital twin in attribute 'href'
```

Comment (mkanal) [reply local/testing/testdata/README.md:147]: or 'href' section

File: local/testing/testdata/README.md [hunk 2]
```diff
@@ -140,8 +140,17 @@ twin with this BPN will be created with an invalid endpointAddress.
...
+
+**--aas3 <[BPN ...] >**  
+(Optional) Flag to create AAS assets in version 3.0
```
Comment (mkanal) [inline local/testing/testdata/README.md:150]:
```suggestion
(Optional) Flag to create AAS assets  according to version 3.0.1
```

File: local/testing/testdata/README.md [hunk 3]
```diff
@@ -140,8 +140,17 @@ twin with this BPN will be created with an invalid endpointAddress.
...
+
+**--allowedBPNs <[BPN ...] >**  
+(Optional) (Required for --aas3) A list of allowed BPNs which will be added to the specificAssetIds when creating AAS Registry assets
```
Comment (mkanal) [inline local/testing/testdata/README.md:153]:
```suggestion
(Optional) (Required for --aas3) A list of allowed BPNs which will be added to the specificAssetIds when creating DTR assets
```

**Review by mkanal — CHANGES_REQUESTED** [review]

File: local/testing/testdata/README.md (hunk 1 above)
Comment (ds-jhartmann) [inline local/testing/testdata/README.md]:
```suggestion
(Optional) If specified, these public datalane URLs will be displayed in the digital twin in attribute 'href'
```

File: local/testing/testdata/README.md (hunk 2 above)
Comment (ds-jhartmann) [inline local/testing/testdata/README.md:150]:
```suggestion
(Optional) Flag to create AAS assets according to version 3.0.1
```

File: local/testing/testdata/README.md [hunk 4]
```diff
@@ -140,8 +140,17 @@ twin with this BPN will be created with an invalid endpointAddress.
 the testdata file and pass the id with this parameter. If nothing is provided, the default policy will be empty with the
 id "default-policy".
 
-**-bpns, --bpns <[BPNS ...] >**  
+**-bpns, --bpns <[BPN ...] >**  
 (Optional) A list of BPNs Filter upload to upload only specific BPNs
 
+**-d, --dataplane <[URL ...] >**  
+(Optional) If specified, these public dataplane URLs will be displayed in the digital twin href
```
Comment (ds-jhartmann) [inline local/testing/testdata/README.md:147]:
```suggestion
(Optional) If specified, these public dataplane URLs will be displayed in the digital twin in attribute 'href'
```

**Review by mkanal — APPROVED** [review]

**Pull request merged**